style/components/_inspect-request.scss

```diff
@@ -7,6 +7,7 @@
   bottom: 0;
   z-index: 2;
   background: rgba($lightgray-light, 0.9);
+  max-height: 60%;
```

Comment:
Overlooked this originally...

`.box-body` will never get larger than 500px, which may constrain `.inspect-request` from ever reaching 60% height, depending on the screen/content/etc.

That'll affect fully maximized windows from ever reaching 60% (probably closer to 50%, depending on content), but when your window isn't full height (say, 400px height), this is what allows `.inspect-request` to hide the close button. With both rules in effect, the 60% on `.inspect-request` would then become the constraint, and `.body-box` may only reach a height of ~250px.

So, I'm going to go ahead and remove the 500px rule, since our intention here is to allow `.inspect-request` to reach up to 60% of the window.